app/integrations/comp_and_pen/scheduled_message_helpers.py

```diff
@@ -178,12 +173,11 @@ def send_comp_and_pen_sms(
                 )
             except Exception as e:
                 current_app.logger.critical(
-                    'Error attempting to send Comp and Pen notification with send_comp_and_pen_sms | '
-                    'record from dynamodb: %s | exception_type: %s - exception: %s',
+                    'Error attempting to send Comp and Pen notification with '
+                    'send_comp_and_pen_sms | record from dynamodb: %s',
                     participant_id,
-                    type(e),
-                    e,
                 )
+                current_app.logger.exception(e)
```

Comment:
Not requesting a change, but just curious: why not just log all this in an exception? I've not seen too many calls to `logger.critical` in the codebase (yet) - is this for datadog dashboards or something?

That makes me wonder - will datadog dashboards which hinge on `logger.error` need to be adjusted since we're changing to `logger.exception`?

Reply:
fair point, do we have any dashboards that rely on logger.error? something we didn't consider when writing the ticket. might be good for @k-macmillan to weigh in.

Reply:
https://docs.python.org/3.10/library/logging.html#logging.exception